### PR TITLE
Add task 5

### PR DESCRIPTION
**Pull request by BenjiSmith1990**

### CODE REVIEW

File: output.txt [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
-         2 | JengARRGGGH! | dexterity | Leslie Scott | grumpy19 | Few games are equiped to fill a player with such a defined sense of mild-peril, but a friendly game of Jenga will turn the mustn't-make-it-fall anxiety all the way up to 11! Fiddly fun for all the family, this game needs little explaination. Whether you're a player who chooses to play it safe, or one who lives life on the edge, eventually the removal of blocks will destabilise the tower and all your Jenga dreams come tumbling down. | https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700 | 2021-01-18 10:01:41.251 |     5
-(1 row)
+    owner     |                        title                         | review_id |    category     |                                        review_img_url                                        |       created_at        | votes |        designer        | count
```
Comment (dappernerddesigns) [inline output.txt:2]: I'd git ignore this, but as it has now been committed you'd need to remove it from github entirely to stop it being tracked.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -83,3 +83,27 @@ describe('GET - /api/reviews/1000000', () => {
...
+                expect(typeof review.review_id).toBe('number')
+                expect(typeof review.category).toBe('string')
+                expect(typeof review.review_img_url).toBe('string')
+                expect(typeof review.created_at).toBe('string')
+                expect(typeof review.votes).toBe('number')
+                expect(typeof review.designer).toBe('string')
+                expect(typeof review.comment_count).toBe('string')
+            })
+        })
+    })
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:108]: There is no test to show that this has been sorted into date order.

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
You're very close on this but you are missing a test for proving the reviews are in date order.

File: output.txt (hunk 1 above)
Comment (BenjiSmith1990) [reply output.txt:2]: that should already be in the ignore file? strange!

File: __tests__/app.test.js (hunk 2 above)
Comment (BenjiSmith1990) [reply __tests__/app.test.js:108]: I put this on my pr request, jest-sorted wouldn't work with the utils?

**Pull request merged**